packages/dd-trace/src/profiling/exporters/agent.js

```diff
@@ -83,7 +89,28 @@ class AgentExporter {
         `profiler_version:${version}`,
         'format:pprof',
         ...Object.entries(tags).map(([key, value]) => `${key}:${value}`)
-      ].join(',')
+      ].join(','),
+      info: {
+        application: {
+          env: this._env,
+          service: this._service,
+          start_time: new Date(perf.nodeTiming.nodeStart + perf.timeOrigin).toISOString(),
+          version: this._appVersion
+        },
+        platform: {
+          hostname: this._host,
+          kernel_name: os.type(),
+          kernel_release: os.release(),
+          kernel_version: os.version()
+        },
+        profiler: {
+          version
+        },
+        runtime: {
+          engine: 'node',
```

Comment:
I don't know, honestly. I think because this is a new value we can choose either. In the predefined values for the top-level `'family'` parameter we're supposed to use `'node'` (see Profiling Intake API internal document.) I wouldn't change runtime tags 'cause they're already used. There's definitely an argument for consistency too. I'm undecided…

Reply:
I decided to change it to `nodejs` to be consistent with the existing tag. `node` is an Intake API internal enum value, not a customer visible value.